Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -393,4 +422,10 @@ namespace Multiplayer
     {
         return PyIsInGameMode();
     }
+
+    void MultiplayerEditorSystemComponent::OnTick(float, AZ::ScriptTimePoint)
+    {
+        m_serverProcessTracePrinter->Pump();
```

Comment:
is it always guaranteed to be not-nullptr here? 
might be worth adding a null-check just in case

Reply:
Fixed! Also added a warning.